### ISSUE TITLE
FSx Lustre max size false positive W1030

### ISSUE THREAD

**Issue by chchang6** [aws-cloudformation/cfn-lint#4706]
### CloudFormation Lint Version

1.56.3

### What operating system are you using?

Mac

### Describe the bug

Linter is throwing a warning about maximum 65536 FSx Lustre size that seems to be a historical artifact that no longer applies.
```bash
chchangu@7cf34dd94b80 templates % cfn-lint w1030-bug.yml
W1030 {'Ref': 'StorageCapacityGiB'} is greater than the maximum of 65536 when 'Ref' is resolved at 'Resources/LustreScratch/Properties/StorageCapacity'
w1030-bug.yml:11:5
```

(AI analysis follows)
  Where 65,536 GiB (64 TiB) came from
  
  At GA and for the early SCRATCH_1 / SCRATCH_2 / PERSISTENT_1 deployment types, the StorageCapacity API/CloudFormation limit was valid values from 1,200 GiB up to a maximum of 65,536 GiB (64 TiB). This cap was documented directly in the FSx API reference and CloudFormation LustreConfiguration docs for those deployment types.
  
  AWS raised the limits over time:
  - PERSISTENT_2 (introduced later) supports much larger file systems — into the hundreds of TiB / PiB range depending on PerUnitStorageThroughput, well beyond 64 TiB.
  - The old 64 TiB cap was specific to the earlier deployment types and their PerUnitStorageThroughput tiers.
  
  Current constraints (they depend on deployment type)
  StorageCapacity valid values today vary by deployment type:
  - PERSISTENT_2 (SSD): 1,200 GiB, then increments of 2,400 GiB, with a much higher ceiling
  
  The authoritative, current values live in the CloudFormation AWS::FSx::FileSystem (https://docs.aws.amazon.com/AWSCloudFormation/latest/TemplateReference/aws-resource-fsx-filesystem.html#cfn-fsx-filesystem-storagecapacity) reference. Lustre can go >> 64 TiB.

### Expected behavior

Clean lint, no warning.

### Reproduction template

```yaml
AWSTemplateFormatVersion: "2010-09-09"

Parameters:
  SubnetId:
    Type: AWS::EC2::Subnet::Id
  StorageCapacityGiB:
    Type: Number
    Description: FSx Lustre storage capacity in GiB. Must be 1200, 2400, or a multiple of 2400.
    Default: 1200
    MaxValue: 460800
    AllowedValues:
      - 1200
      - 460800

Resources:
  LustreScratch:
    Type: AWS::FSx::FileSystem
    Properties:
      FileSystemType: LUSTRE
      FileSystemTypeVersion: "2.15"
      LustreConfiguration:
        DataCompressionType: LZ4
        DeploymentType: PERSISTENT_2
        PerUnitStorageThroughput: 125
        MetadataConfiguration:
          Mode: AUTOMATIC
      StorageCapacity: !Ref StorageCapacityGiB
      StorageType: SSD
      SubnetIds:
        - !Ref SubnetId
```
